io/src/vtk_lib_io.cpp

```diff
@@ -240,6 +240,11 @@ pcl::io::vtk2mesh (const vtkSmartPointer<vtkPolyData>& poly_data, pcl::PolygonMe
   mesh.cloud.width = mesh.cloud.height = 0;
   mesh.cloud.is_dense = true;
 
+  if (poly_data->GetPoints () == nullptr)
```

Comment:
Should we even resize the mesh? Shouldn't we perform the check as the first thing in the function? The use of 0 for return value can be used at call-sites to check if the call failed.

Reply:
I think it is safer to initialize `mesh` first. If there was something else in `mesh` which is not overwritten, then this might lead to confusion.

Reply:
`mesh` is a reference, so it might be safe to do nothing, but better safe than sorry